--- Net_Nmap-0.9.9/usr/share/pear/Net/Nmap.php~	2008-05-18 05:07:01.363277004 +0200
+++ Net_Nmap-0.9.9/usr/share/pear/Net/Nmap.php	2008-05-18 05:08:17.223570225 +0200
@@ -29,8 +29,8 @@
  */
 
 require_once 'System.php';
-require_once 'Nmap/Parser.php';
-require_once 'Nmap/Exception.php';
+require_once 'Net/Nmap/Parser.php';
+require_once 'Net/Nmap/Exception.php';
 
 /**
  * Simple wrapper interface for the Nmap utility.
